### ISSUE TITLE
The performance of the algorithm in tff  is much lower

### ISSUE THREAD

**Issue by mlchuxueing** [tensorflow/federated#470]
Hello，
When I use the logistic regression algorithm to perform the two classification task on tff, the performance of the training is reduced compared to using only the logistic regression algorithm. What is the reason? I just divided a large data set into several small data sets. They have the same feature. Below is my result. Can you tell me why?
in tff:  accuracy=0.55
only logisitic regression:  accuracy=0.71

**Comment by ZacharyGarrett** [comment]
This question likely is better suited for StackOverflow, possibly with the [`tensorflow-federated` tag](https://stackoverflow.com/questions/tagged/tensorflow-federated); there are a large number of factors that could be contributing to this behavior.

In the StackOverflow question I would suggest providing additional details (showing actual code can also help), including details about: how were the small data sets created? what hyperparameters and optimizer were used? how many features vs training examples are in the dataset? how many epochs are run locally on the clients before sending a model update back?

 I'm going to close this issue, could you please ask the question at StackOverflow?
